backend/contentstore/s3store.go

```diff
@@ -70,6 +71,23 @@ func (s *S3Store) Read(key string) (io.Reader, error) {
 	return res.Body, nil
 }
 
+func (s *S3Store) SendURL(key string) (*string, error) {
+	contentType := "image/jpeg"
+	req, _ := s.s3Client.GetObjectRequest(&s3.GetObjectInput{
+		Bucket:              aws.String(s.bucketName),
+		Key:                 aws.String(key),
+		ResponseContentType: aws.String(contentType),
+	})
+
+	// TODO TN how long should this be? and Should I try to cache these images
+	url, err := req.Presign(time.Hour * 24)
```

Comment:
how long do you guys think these images should persist? I can't imagine someone leaving the application on the evidence page for more than an hour or so?